taichi/transforms/offload.cpp

```diff
@@ -77,7 +77,9 @@ class Offloader {
           offloaded_ranges.end_stmts.insert(
               std::make_pair(offloaded.get(), s->end));
         }
-        offloaded->num_cpu_threads = s->parallelize;
+        offloaded->num_cpu_threads =
+            std::min(s->parallelize,
```

Comment:
Btw do we have a decorator to tweak s->parallelize?

Reply:
We do :-) `ti.parallelize(num_threads)` can change the number of threads of the following for loop. The API is not yet stable so it's undocumented.